CHANGELOG.md

```diff
@@ -12,6 +12,323 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 20.5 (2020-04-10)
+
+### Process
+
+- Include oasis-core-runtime-loader and oasis-net-runner in releases
+  ([#2780](https://github.com/oasislabs/oasis-core/issues/2780))
+
+### Removals and Breaking changes
+
+- Sentry nodes no longer require TLS certificate file of the upstream node
```

Comment:
I think no need for quesnet docs changes as this was only needed for grpc sentries and is not mentioned in the docs (should double-check)

Reply:
Yeah could be confusing, removed.